test/README.md

```diff
@@ -1,15 +1,13 @@
 
 # Resonate API Testing
 
-These are tests for the WIP Resonate API (v4) migration. This iteration of the tests development simply migrates tests as-built to a Mocha environment and begins development of new tests of the WIP api specification to-be. This stage of the testing effort deliberately keeps things simple. Later iterations will focus on more fancy stuff.
-
-Some clarification is needed regarding the test target. There are two possible targets:
-* http://localhost:4000/api/v3 (this repo: api(v4))
-* https://stream.resonate.coop/api/v3/ (beam/api)
+These are tests for the WIP Resonate API (v4) migration. This iteration begins development of tests of the WIP api specification. This stage of the testing effort deliberately keeps things simple. Perhaps later iterations will focus on more fancy stuff.
 
 The purpose of these tests is to verify / validate the endpoints of this API, and to begin development of a more robust API test suite.
 
-For the sake of total clarity, these tests live in the `/api/test` directory. The previous tests live in `/api/test/as-built` directory. The new tests live in `api/test/to-be` directory. In the `api/test/to-be` directory there are three subfolders (Admin.ts, Api.ts, and User.ts), each of which corresponds to a file in the `beam/src/services` folder.
+These tests reference the [beam](https://github.com/resonatecoop/beam/tree/main/src/services) repository. The 'beam' repo serves as a reference for available endpoints, which are to be integrated into the api(v4) repo.
+
+The api(v4) tests live in the `/api/test` directory. In this directory there are three subfolders (`Admin.ts`, `Api.ts`, and `User.ts`), each of which corresponds to a file in the [beam](https://github.com/resonatecoop/beam/tree/main/src/services) repository.
```

Comment:
```suggestion
The api(v4) tests live in the `./test` directory. In this directory there are three subfolders (`Admin.ts`, `Api.ts`, and `User.ts`), each of which corresponds to a file in the [beam](https://github.com/resonatecoop/beam/tree/main/src/services) repository.
```

The `api` refers to the repository folder, right? If the user is navigating from the main folder as root (they could arguably have called their folder something else, like for me it's called `etanoser`)